caffe2/serialize/inline_container_test.cc

```diff
@@ -0,0 +1,64 @@
+#include <cstdio>
+#include <iostream>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "caffe2/serialize/inline_container.h"
+
+namespace at {
+namespace {
+
+TEST(PyTorchFileWriterAndReader, SaveAndLoad) {
+  int64_t kFieldAlignment = 64L;
+  // create a name for temporary file
+  // TODO to have different implementation for Windows and POXIS
+  std::string tmp_name = std::tmpnam(nullptr);
```

Comment:
delete file in the end (I wish we could use folly for this)

Reply:
Yeah, I should always delete it. Will add it.